subprojects/build.gradle.kts

```diff
@@ -400,6 +400,7 @@ fun Project.configureJunit5Tests() {
     tasks.withType<Test> {
         useJUnitPlatform()
         maxParallelForks = 8
+        setForkEvery(1)
```

Comment:
By default, the Test task doesn't fork the execution process
Now on each Test class, we will start a new process with the maximum amount equals to 8

Reply:
Nice,
check memory consumption though, because i did no tuning here and gradle test kit inherit daemon settings by default

Reply:
Why do we need a new process for each test? Do we have any problems?

Reply:
Check locally it changes nothing
Removed